lightfm-port/main.py

```
@@ -0,0 +1,26 @@
+data = fetch_movielens(min_rating=5.0)
+
+# Instantiate and train the model with fixed random state
+model = LightFM(loss='warp', random_state=42)
+model.fit(data['train'], epochs=30, num_threads=2)
+
+# Evaluate the trained model
+test_precision = precision_at_k(model, data['test'], k=5).mean()
+
+# Print results for comparison with PHP version
+print(f"Test precision@5: {test_precision:.6f}")
+print(f"Model parameters shape:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)